hubblestack/audit/command_line_parser.py

```diff
@@ -457,3 +457,22 @@ def get_filtered_params_to_log(block_id, block_dict, extra_args=None):
         'key_aliases': key_aliases,
         'delimiter': delimiter
     }
+
+
+def get_failure_reason(block_id, block_dict, extra_args=None):
+    """
+
```

Comment:
Also please add a one-line about this function (what it does) here.

Reply:
Done